ui/main.qml

```diff
@@ -369,7 +386,10 @@ StatusWindow {
 
     Loader {
         id: startupOnboardingLoader
+
         anchors.fill: parent
+        anchors.topMargin: Constants.isMobile ? parent.SafeArea.margins.top : 0
```

Comment:
I wouldn't make this distinction; desktop OSs also have these safe areas and even if it might not be implemented at this point, it will be zero anyway

Reply:
without this on Macos it looks like this 

<img width="1275" alt="Screenshot 2025-07-08 at 09 55 59" src="https://github.com/user-attachments/assets/7cd65ba4-7282-4c3f-a8c4-c93ac9d3f2b1" />

the title bar is kept separated from our app ui. Perhaps we correctly use ToolBar and header in the future instead, it might be handled properly and hence this check.

Reply:
Yeah, ToolBar should be aware of these